content/events/summit-brazil-24/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "[Upcoming Event] Espressif Summit Brazil 2024"
+date: 2024-06-10
+draft: false
+description: "Espressif Systems Summit in Brazil"
+tags: ["event", "announcement"]
+showDate : false
+---
+
+## Espressif Summit Brazil 2024
+
+We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm. On the first day, we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
```

Comment:
Suggest changing the date and time to improve readability, maybe like this:

from July 17 to 18, 9 am - 6 pm.

We have a [chapter](http://mos.espressif.com/time-and-dates.html) in our Espressif Manual of Style about it, but the guidelines are quite flexible.